### PR TITLE
ci: use razorops v2 spec

### PR DESCRIPTION
**Pull request by dsyoi**
@srz-zumix Updated CI spec to use v2 version of [Razorops](https://docs.razorops.com/config/spec/)

### CODE REVIEW

File: .razorops.yaml [hunk 1]
```diff
@@ -1,58 +1,27 @@
...
-  # centos7:
-  #   image: centos/devtoolset-7-toolchain-centos7:latest
-  #   commands:
-  #     - cd test
-  #     - make clean
-  #     - make
-  #     - make test
-  #     - make clean
-  #   when: *condition
+  lint-editorconfig:
```
Comment (srz-zumix) [inline .razorops.yaml:15]: 👍

File: .razorops.yaml [hunk 2]
```diff
@@ -1,58 +1,27 @@
...
 
-  # gcc3-centos5:
-  #   image: srzzumix/gcc3:centos5
-  #   commands:
-  #     - find . -type f \( -name \*.hpp -or -name \*.cpp -or -name \*.ipp \) -exec sed -i -e '1s/^\xEF\xBB\xBF//' {} \;
-  #     - make -C test
-  #     - make -C test test
-  #   when: *condition
+trigger:
+  when: branch in ('master', 'develop') || branch =~ 'razorops.*'
```
Comment (srz-zumix) [inline .razorops.yaml:27]: RazorOps doesn't seem to be running.
The condition of Triggers seems to use if.

https://docs.razorops.com/pipeline/triggers/

Comment (dsyoi) [reply .razorops.yaml:27]: I guess the syntax is correct, but Razorops doesn't support builds from forked repos atm.

**Comment by srz-zumix** [comment]
@dsyoi 
I would like to add the lint-editorconfig task. Can you please resolve the conflict?

**Review by srz-zumix — APPROVED** [review]
LGTM
Thank you!

**Pull request merged**